README.md

```diff
@@ -151,35 +151,59 @@ flowchart TD;
 
 With _DLlib_, you can write distributed deep learning applications as standard (**Scala** or **Python**) Spark programs, using the same *Spark DataFrame* and *ML Pipeline* APIs.
 
-<details><summary>Show DLlib example</summary>
+<details><summary>Show DLlib Scala example</summary>
 <br/>
 
-You can build distributed deep learning applications for Spark using *DLlib* in 3 simple steps:
+Image Classification using *DLlib* Keras-Style Scala API.
 
 ```scala
-// 1. Call `initNNContext` at the beginning of the code: 
-import com.intel.analytics.bigdl.dllib.NNContext
+// 1. Init the context for DLlib
 val sc = NNContext.initNNContext()
```

Comment:
some examples and user guide still uses ```Engine.init``` for initialization. Checked with Kai should all replace with `NNContext.init`. Maybe dingding could also have a look.